### DISCUSSION TITLE
configure type: "asset/resource"seem unable to export the jpg|png|ico URL from an html file.

### DISCUSSION THREAD

**Discussion by Tiny-Wei** [webpack/webpack#15116] [Q&A]
## Feature request

**What is the expected behavior?**
Use the following configuration，It seem unable to export the jpg|png|ico URL from an html file.
{
        test: /\.(jpg|png|ico)$/,
        type: "asset/resource",
        generator: {
          filename: "img/[hash][ext][query]"
        },
}

**What is motivation or use case for adding/changing the behavior?**
I don’t have to configuring additional loaders.

**How should this be implemented in your opinion?**
i don't know

**Are you willing to work on this yourself?**
yes,But my current level is not enough

**Comment by alexander-akait** [comment]
Please clarify your problem
